app/client/head.html

```diff
@@ -22,4 +22,5 @@
   <meta name="msapplication-TileImage" content="/favicons/mstile-144x144.png">
   <meta name="msapplication-config" content="/favicons/browserconfig.xml">
   <meta name="theme-color" content="#ffffff">
+  <script type="text/javascript" src="https://sandstorm.io/stats.js" async></script>
```

Comment:
Yeah, I figure we can leave this async until such time as we actually have experiments to run, and then we'll need to make it sync to avoid FOUC.